mmcv/cnn/rfsearch/operator.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+from typing import Dict
+
+import numpy as np
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+from mmcv.runner import BaseModule
+from mmcv.utils.logging import get_logger
+from .utils import expands_rate
+
+logger = get_logger('Operators')
+
+
+class ConvRFSearchOp(BaseModule):
+    """Based class of ConvRFSearchOp.
+
+    Args:
+        op_layer (nn.Module): pytorch module, e,g, Conv2d
+        global_config (Dict): config dict. Defaults to None.
+    """
+
+    def __init__(self, op_layer: nn.Module, global_config: Dict = {}):
```

Comment:
The default value is not consistent with docstring. The only acceptable value is 'absavg' so should make it as default value and check it in __init__?